util/oidc/oidc.go

```diff
@@ -148,32 +139,99 @@ func (a *ClientApp) oauth2Config(scopes []string) (*oauth2.Config, error) {
 	}, nil
 }
 
+func encrypt(data []byte, passphrase string) ([]byte, error) {
```

Comment:
What do you think about having the encrypt/decrypt as global utility functions that could be reused in different parts of the application?

Reply:
Agree. Moving it to `util/crypto`

Reply:
done